neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -21,75 +23,11 @@ public abstract class Nep5Token<TState> : NativeContract
         protected const byte Prefix_TotalSupply = 11;
         protected const byte Prefix_Account = 20;
 
-        protected Nep5Token() : base()
+        protected Nep5Token()
```

Comment:
We should call the base contructor
```suggestion
        protected Nep5Token() : base()
```